mlflow/server/js/src/utils/Utils.js

```diff
@@ -32,6 +32,7 @@ class Utils {
   static sourceTypeTag = 'mlflow.source.type';
   static gitCommitTag = 'mlflow.source.git.commit';
   static entryPointTag = 'mlflow.project.entryPoint';
+  static userTag = 'mlflow.user';
```

Comment:
Wonder if there is a way to have a single source of truth for all these constants.

Reply:
Yeah, not sure. If we could define string constants in proto, maybe we could refer to them in all generated languages?